.github/workflows/pipeline_building.yml

```diff
@@ -0,0 +1,19 @@
+name: Pipeline building
+
+on: push
+
+jobs:
+  preparation:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Build
+        run: g++ -std=c++17 -o a.out *.cpp
+      - name: Regaining right
+        run: chmod a+x ./battle_prep.sh
+      - name: Fixing line endings failure
+        run: sed -i -e 's/\r$//' ./battle_prep.sh
```

Comment:
what? ezt azert ne. hogy a scripteteteket editaljatok GHA-ban. Tessek normalis line endinggel elmenteni. bash scriptet gondolom linux subsystem alol futtattok windowson is, es gondolom az is normal linux line endinget hasznal.